entity-framework/core/modeling/relationships.md

```diff
@@ -164,7 +161,7 @@ public class Post
 
 ### Cascade Delete
 
-By convention, cascade delete will be set to *Cascade* for required relationships and *SetNull* for optional relationships. *Cascade* means dependent entities are also deleted. *SetNull* means that foreign key properties in dependent entities are set to null.
+By convention, cascade delete will be set to *Cascade* for required relationships and *Restrict* for optional relationships. *Cascade* means dependent entities are also deleted. *Restrict* means that dependent entities remain unchanged and must be manually deleted, or updated to point to a valid principal entity.
```

Comment:
Why explaining only half of the story and not both, tracked and untracked?
Perhaps something like this:
When dependent entities are untracked, Restrict will leave them unchanged. This means that they must be manually deleted, or their foreign key properties must be updated to null or made to point to a valid principal entity. For tracked entities, EF will always attempt to set the FK properties to null.